sql/uniswap_v2/src/lib.rs

```diff
@@ -0,0 +1,307 @@
+mod abi;
```

Comment:
Do we need to version-control the rust code? If they are all autogen'ed and deterministic, why not just generate in the build stage?